experimental/rubyfmt_c_main/BUILD

```diff
@@ -0,0 +1,21 @@
+cc_binary(
```

Comment:
Is this binary here just to test the build? i.e., do we expect to replace this `cc_binary` with a library rule eventually when integrating it into sorbet's LSP?

Reply:
yes, eventually this would just end up as part of Sorbet's build, but this makes sure everything is working.